test.py

```diff
@@ -20,23 +21,30 @@ class TestMain(unittest.TestCase):
     def test_disposition_notification(self):
         mail = get_email_as_bytes('disposition-notification.eml')
         body = serialize_mail(mail)
+        body_map = {k:v for k, v in body}
```

Comment:
Czy rozumiesz dlaczego potrzebujemy `body_map`? Tak było napisane w `v2`, ale nie rozumiem celu i chyba przed wejściem na master można to sprzątnąć.

Reply:
Też mnie to zastanowiło, ale nie chciałem już wnikać, żeby się nie rozpraszać. Kod jest też kiepsko sformatowany, więc może warto to sprzątanie przenieść do osobnego tasku i poszerzyć o generalny refactoring i reformat kodu?

Reply:
OK